automation/project/Settings.scala

```diff
@@ -38,18 +38,18 @@ object Settings {
   )
 
   //common settings for all sbt subprojects
-  val commonSettings =
+  val commonSettings: Seq[Def.Setting[_]] =
```

Comment:
IMO this particular inspection is busy-work, so i wouldn't stress too much about adding type annotations for everything